### ISSUE TITLE
TypeError: 'NoneType' object is not callable

### ISSUE THREAD

**Issue by lovekesh-thakur** [conda/cookiecutter-conda-python#19]
While building the package using 'conda build' command,  i am getting error as below:

Traceback (most recent call last):
  File "/home/lovekesh/anaconda3/bin/conda-build", line 11, in <module>
    sys.exit(main())
  File "/home/lovekesh/anaconda3/lib/python3.7/site-packages/conda_build/cli/main_build.py", line 469, in main
    execute(sys.argv[1:])
  File "/home/lovekesh/anaconda3/lib/python3.7/site-packages/conda_build/cli/main_build.py", line 460, in execute
    verify=args.verify, variants=args.variants)
  File "/home/lovekesh/anaconda3/lib/python3.7/site-packages/conda_build/api.py", line 209, in build
    notest=notest, need_source_download=need_source_download, variants=variants)
  File "/home/lovekesh/anaconda3/lib/python3.7/site-packages/conda_build/build.py", line 2322, in build_tree
    bypass_env_check=True)
  File "/home/lovekesh/anaconda3/lib/python3.7/site-packages/conda_build/render.py", line 785, in render_recipe
    m = MetaData(recipe_dir, config=config)
  File "/home/lovekesh/anaconda3/lib/python3.7/site-packages/conda_build/metadata.py", line 863, in __init__
    self.parse_again(permit_undefined_jinja=True, allow_no_other_outputs=True)
  File "/home/lovekesh/anaconda3/lib/python3.7/site-packages/conda_build/metadata.py", line 940, in parse_again
    bypass_env_check=bypass_env_check),
  File "/home/lovekesh/anaconda3/lib/python3.7/site-packages/conda_build/metadata.py", line 1528, in _get_contents
    rendered = template.render(environment=env)
  File "/home/lovekesh/anaconda3/lib/python3.7/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/home/lovekesh/anaconda3/lib/python3.7/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/home/lovekesh/anaconda3/lib/python3.7/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/home/lovekesh/repository_name/conda.recipe/meta.yaml", line 45, in top-level template code
    requires:
TypeError: 'NoneType' object is not callable

Here's a list of my installed packages.

# packages in environment at /home/lovekesh/.conda/envs/test-env:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       0_gnu    conda-forge
arrow                     0.12.1                   py35_2    conda-forge
asn1crypto                0.24.0                   py35_3    conda-forge
beautifulsoup4            4.6.3                    py35_0    conda-forge
binaryornot               0.4.4                      py_1    conda-forge
bzip2                     1.0.8                h516909a_2    conda-forge
ca-certificates           2019.11.28           hecc5488_0    conda-forge
certifi                   2018.8.24             py35_1001    conda-forge
cffi                      1.11.5           py35h5e8e0c9_1    conda-forge
chardet                   3.0.4                    py35_3    conda-forge
click                     7.1.1              pyh8c360ce_0    conda-forge
conda                     4.5.11                   py35_0    conda-forge
conda-build               3.15.1                   py35_0    conda-forge
conda-env                 2.6.0                         1    conda-forge
cookiecutter              1.5.1                    py35_0    conda-forge
cryptography              2.3.1            py35hdffb7b8_0    conda-forge
cryptography-vectors      2.3.1                    py35_0    conda-forge
filelock                  3.0.10                     py_0    conda-forge
future                    0.16.0                   py35_2    conda-forge
glob2                     0.7                        py_0    conda-forge
idna                      2.9                        py_1    conda-forge
jinja2                    2.11.1                     py_0    conda-forge
jinja2-time               0.2.0                      py_2    conda-forge
libffi                    3.2.1             he1b5a44_1007    conda-forge
libgcc-ng                 9.2.0                h24d8f2e_2    conda-forge
libgomp                   9.2.0                h24d8f2e_2    conda-forge
libstdcxx-ng              9.2.0                hdf63c60_2    conda-forge
markupsafe                1.0              py35h470a237_1    conda-forge
ncurses                   6.1               hf484d3e_1002    conda-forge
openssl                   1.0.2u               h516909a_0    conda-forge
patchelf                  0.10                 he1b5a44_0    conda-forge
pip                       20.0.2                     py_2    conda-forge
pkginfo                   1.5.0.1                    py_0    conda-forge
poyo                      0.5.0                      py_0    conda-forge
psutil                    5.4.7           py35h14c3975_1001    conda-forge
pycosat                   0.6.3           py35h14c3975_1001    conda-forge
pycparser                 2.20                       py_0    conda-forge
pyopenssl                 18.0.0                   py35_0    conda-forge
python                    3.5.5                h5001a0f_2    conda-forge
python-dateutil           2.8.1                      py_0    conda-forge
pytz                      2019.3                     py_0    conda-forge
pyyaml                    3.13             py35h470a237_1    conda-forge
readline                  7.0               hf8c457e_1001    conda-forge
requests                  2.13.0                   py35_0    conda-forge
ruamel_yaml               0.15.71          py35h470a237_0    conda-forge
setuptools                40.4.3                   py35_0    conda-forge
six                       1.14.0                     py_1    conda-forge
sqlite                    3.28.0               h8b20d00_0    conda-forge
tk                        8.6.10               hed695b0_0    conda-forge
tqdm                      4.44.1             pyh9f0ad1d_0    conda-forge
wheel                     0.34.2                     py_1    conda-forge
whichcraft                0.6.1                      py_0    conda-forge
xz                        5.2.4             h516909a_1002    conda-forge
yaml                      0.1.7             h14c3975_1001    conda-forge
zlib                      1.2.11            h516909a_1006    conda-forge

Thanks

**Comment by luyang93** [comment]
I also come to the same problem.I have no idea to deal with it.

**Comment by luyang93** [comment]
I revert to 06fa9b933774ac1b9a83b009f182e73f418ebc5f.
It works as well.

**Comment by luyang93** [comment]
solved in https://github.com/conda/cookiecutter-conda-python/pull/21

**Comment by NEGU93** [comment]
I had the same problem:
```
No numpy version specified in conda_build_config.yaml.  Falling back to default numpy value of 1.11
WARNING:conda_build.metadata:No numpy version specified in conda_build_config.yaml.  Falling back to default numpy value of 1.11
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\Scripts\conda-build-script.py", line 10, in <module>
    sys.exit(main())
  File "C:\ProgramData\Anaconda3\lib\site-packages\conda_build\cli\main_build.py", line 445, in main
    execute(sys.argv[1:])
  File "C:\ProgramData\Anaconda3\lib\site-packages\conda_build\cli\main_build.py", line 436, in execute
    verify=args.verify, variants=args.variants)
  File "C:\ProgramData\Anaconda3\lib\site-packages\conda_build\api.py", line 209, in build
    notest=notest, need_source_download=need_source_download, variants=variants)
  File "C:\ProgramData\Anaconda3\lib\site-packages\conda_build\build.py", line 2302, in build_tree
    bypass_env_check=True)
  File "C:\ProgramData\Anaconda3\lib\site-packages\conda_build\render.py", line 786, in render_recipe
    m = MetaData(recipe_dir, config=config)
  File "C:\ProgramData\Anaconda3\lib\site-packages\conda_build\metadata.py", line 862, in __init__
    self.parse_again(permit_undefined_jinja=True, allow_no_other_outputs=True)
  File "C:\ProgramData\Anaconda3\lib\site-packages\conda_build\metadata.py", line 939, in parse_again
    bypass_env_check=bypass_env_check),
  File "C:\ProgramData\Anaconda3\lib\site-packages\conda_build\metadata.py", line 1527, in _get_contents
    rendered = template.render(environment=env)
  File "C:\ProgramData\Anaconda3\lib\site-packages\jinja2\asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "C:\ProgramData\Anaconda3\lib\site-packages\jinja2\environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "C:\ProgramData\Anaconda3\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "C:\ProgramData\Anaconda3\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "W:\HardDiskDrive\Documentos\GitHub\cvnn\conda.recipe\meta.yaml", line 45, in top-level template code
    requires:
TypeError: 'NoneType' object is not callable
```

As explained by @luyang93, the solution was to change the file `conda.recipe/meta.yaml` two lines as it shows [here](https://github.com/conda/cookiecutter-conda-python/pull/21/files)

**Comment by hellmersjl3ds** [comment]
It looks like those changes at https://github.com/conda/cookiecutter-conda-python/pull/21/files haven't been merged into master yet.  Any chance of that happening soon?

**Comment by AlbertDeFusco** [comment]
This has been fixed by #23
